syntax = "proto3";

message Request {
    string data = 1;
}

message Response {
    string data = 1;
}

service Datastore {
   rpc RegisterClient(Request) returns (Response) {}
   rpc get(Request) returns (Response) {}
}
